.github/actions/generic-ci/action.yml

```diff
@@ -25,10 +25,6 @@ inputs:
     description: 'Label to control bundle'
     required: false
     default: 'no-bundle'
-  egl_label:
-    description: 'Label to control egl'
-    required: false
-    default: 'no-egl'
```

Comment:
i think its better to keep this egl_label to make sure Xvfb is not used at all when running headless ci